.github/workflows/ci.yml

```diff
@@ -19,25 +19,22 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - name: GitHub Action for SwiftLint
-        uses: norio-nomura/action-swiftlint@3.1.0
-      - name: GitHub Action for SwiftLint with --strict
-        uses: norio-nomura/action-swiftlint@3.1.0
-        with:
-          args: --strict
+        uses: norio-nomura/action-swiftlint@3.2.1
       - name: GitHub Action for SwiftLint (Only files changed in the PR)
-        uses: norio-nomura/action-swiftlint@3.1.0
+        uses: norio-nomura/action-swiftlint@3.2.1
         env:
           DIFF_BASE: ${{ github.base_ref }}
```

Comment:
I'm confused why we're running `action-swiftlint` 2 times (previous 3 times)

Reply:
Thanks, I have removed the only files changed in the PR. It was not working anyhow because of dubious ownership problem.